src/test/pythonnode_test.cpp

```diff
@@ -479,3 +442,404 @@ TEST_F(MediapipePythonNodeTest, PythonNodePassArgumentsToConstructor) {
+}
+
+
+std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>> prepareInputSidePacket(const std::string& handlerPath, PythonBackend * pythonBackend) {
+    // Create side packets
+    auto fsHandlerPath = std::filesystem::path(handlerPath);
+    fsHandlerPath.replace_extension();
+
+    std::string parentPath = fsHandlerPath.parent_path();
+    std::string filename = fsHandlerPath.filename();
+
+    py::gil_scoped_acquire acquire;
+    py::module_ sys = py::module_::import("sys");
+    sys.attr("path").attr("append")(parentPath.c_str());
+    py::module_ script = py::module_::import(filename.c_str());
+    py::object OvmsPythonModel = script.attr("OvmsPythonModel");
+    py::object pythonModel = OvmsPythonModel();
+
+    std::shared_ptr<PythonNodeResource> nodeResource = std::make_shared<PythonNodeResource>(pythonBackend);
+    nodeResource->nodeResourceObject = std::make_unique<py::object>(pythonModel);
+
+    std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>> nodesResources {{"pythonNode", nodeResource}};
+    return nodesResources;
+}
+
+TEST_F(PythonFlowTest, PythonCalculatorTestSingleInSingleOut) {
+    std::string testPbtxt = R"(
+        calculator: "PythonExecutorCalculator"
+        name: "pythonNode"
+        input_side_packet: "PYTHON_NODE_RESOURCES:py"
+        input_stream: "INPUT:in"
+        output_stream: "OUTPUT:out"
+        options: {
+            [mediapipe.PythonExecutorCalculatorOptions.ext]: {
+                handler_path: "/ovms/src/test/mediapipe/python/scripts/symmetric_increment.py"
+            }
+        }
+    )";
+
+    mediapipe::CalculatorRunner runner(testPbtxt);
+    py::gil_scoped_acquire acquire;
+    try {
+        std::string handlerPath = "/ovms/src/test/mediapipe/python/scripts/symmetric_increment.py";
+        // Create side packets
+        std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>> nodesResources = prepareInputSidePacket(handlerPath, getPythonBackend());
+        addInputSidePacket("PYTHON_NODE_RESOURCES", nodesResources, 0, &runner);
+
+        // Prepare inputs
+        std::string datatype = "FP32";
+        std::string inputName = "INPUT";
+        int numElements = 3;
+        float input1[] = {1.0, 1.0, 1.0};
+        SimpleTensor<float> tensor1 = SimpleTensor<float>::createTensor(inputName, input1, datatype, numElements);
+        addInputItem(inputName, tensor1.pyTensor, 0, &runner);
+
+
+        // Run calculator
+        {
+            py::gil_scoped_release release;
+            ASSERT_EQ(runner.Run(), absl::OkStatus());
+        }
+
+        // Read and check outputs
+        std::vector<float> output1 = SimpleTensor<float>::readVectorFromOutput("OUTPUT", tensor1.numElements, &runner);
+        EXPECT_EQ(output1, tensor1.getIncrementedVector());
+
+    } catch (const pybind11::error_already_set& e) {
+        std::cout << e.what() << std::endl;
+        ASSERT_EQ(1,0);
```

Comment:
```suggestion
        ASSERT_EQ(1,0) << e.what();
```
This way you do not need iostream

Reply:
Will do